### ISSUE TITLE
[java] `TooManyStaticImports` triggered with explicit warning suppression

### ISSUE THREAD

**Issue by Ledmington** [pmd/pmd#6983]
**Affects PMD Version:**

7.26.0

**Rule:**

[TooManyStaticImports](https://docs.pmd-code.org/latest/pmd_rules_java_codestyle.html#toomanystaticimports)

**Description:**

If I create a class with too many static imports the rule gets triggered, but when I explicitly suppress it with `@SuppressWarnings("PMD.TooManyStaticImports")` it should not be triggered but it does.

**Code Sample demonstrating the issue:**

```java
import static utils.BitUtils.and;
import static utils.BitUtils.asByte;
import static utils.BitUtils.asInt;
import static utils.BitUtils.asShort;
import static utils.BitUtils.not;
import static utils.BitUtils.or;
import static utils.BitUtils.shr;

@SuppressWarnings("PMD.TooManyStaticImports")
class Example {
	public Example() {
		// ...
	}
}
```

**Running PMD through:** Gradle
